NitroxClient/GameLogic/EscapePodManager.cs

```diff
@@ -1,45 +1,34 @@
 ﻿using System.Collections.Generic;
 using NitroxClient.Communication.Abstract;
-using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.GameLogic.Helper;
 using NitroxClient.Unity.Helper;
 using NitroxModel.DataStructures.GameLogic;
 using NitroxModel.Logger;
 using NitroxModel.Packets;
 using UnityEngine;
 
-namespace NitroxClient.Communication.Packets.Processors
+namespace NitroxClient.GameLogic
 {
-    public class BroadcastEscapePodsProcessor : ClientPacketProcessor<BroadcastEscapePods>
+    public class EscapePodManager
     {
-        public static bool SURPRESS_ESCAPE_POD_AWAKE_METHOD;
-
         private readonly IPacketSender packetSender;
-        private IMultiplayerSession multiplayerSession;
+        private readonly IMultiplayerSession multiplayerSession;
+
+        public static bool SURPRESS_ESCAPE_POD_AWAKE_METHOD;
         private readonly Vector3 playerSpawnRelativeToEscapePodPosition = new Vector3(0.9f, 2.1f, 0);
         private readonly Dictionary<string, GameObject> escapePodsByGuid = new Dictionary<string, GameObject>();
 
-        private string myEscapePodGuid;
+        public string MyEscapePodGuid;
 
-        public BroadcastEscapePodsProcessor(IPacketSender packetSender, IMultiplayerSession multiplayerSession)
+        public EscapePodManager(IPacketSender packetSender, IMultiplayerSession multiplayerSession)
         {
             this.packetSender = packetSender;
             this.multiplayerSession = multiplayerSession;
         }
 
-        public override void Process(BroadcastEscapePods packet)
-        {
-            if (myEscapePodGuid == null)
-            {
-                AssignPlayerToEscapePod(packet);
-            }
-
-            SyncEscapePodGuids(packet);
-        }
-
-        private void AssignPlayerToEscapePod(BroadcastEscapePods packet)
+        public void AssignPlayerToEscapePod(List<EscapePodModel> escapePods)
```

Comment:
This method signature is a little odd to me.  `escapePodManager.AssignPlayerToEscapePod(escapePods). ` ideally this would simply take a guid.  Maybe we can have the assigned escape pod on the initial sync playerData so that we can simply pass it in.  This would require you to probably make a change in the server to do:
```String escapePodGuid = escapePodManager.assignPlayerToEscapePod(player, out newlyCreatedPod);```